### DISCUSSION TITLE
Tell us about how you use PhotoPrism!

### DISCUSSION THREAD

**Discussion by graciousgrey** [photoprism/photoprism#706] [Show and tell]
We are interested in getting to know our community and the way you use PhotoPrism better ✨ 
What infrastructure are you using? 
How many photos/videos do you manage with it? 
Are you a professional photographer or is it your hobby? 
What do you like most about the app?

Feel free to comment below 💫 ❤️ 💫

**Comment by tootbrute** [comment]
1. Self-hosting on Lenovo Q190 (1.6GHZ, 8 GB RAM), using Yunohost.org self hosting package on Debian 10
2. Current photo collection size: 48,000 photos.
3. Not a professional photographer. I use this as a Google Photos replacement. I want my photos to be organized, and easily accessible.
4. I'm new to using this, but I like the organization. It organizes things by date, and can automate things like creating albums for places, and dates. I want my photos backed up, but want to easily grab photos by date, name, or location when I need them (without giving up my data to Google or other third parties)

**Reply by rickysarraf** [reply]
1. Self-hosted on an laptop (with a broken display) running on Debian Testing
2. Photo collection is currently at 11,000 photos/videos
3. Not a professional photographer. I want a free tool for my private life.
4. I love the fact that PhotoPrism is free software and I feel much confident in investing my time using it to organize a part of my digital life.
5. I plan to use a combination of Digikam (for mass annotation and metadata writing) and PhotoPrism (for presentation of the data)
6. Would love to see a PhotoPrism mobile app so that I can completely get rid of Google Photos
7. Currently, I use Android mobile backup tools to auto-move pictures from my phone to PhotoPrism server share
8. It would be nice to see PhotoPrism, much more, enhance its TensorFlow results. I'd like to see PhotoPrism make recommendations on the basis of: **This {Day, Week, Month, Year, Random, Then n' Now, Moments}**, something along the lines of what Google Photos does.

**Comment by alexisLefebvre** [comment]
1. PhotoPrism runs on Docker on a server with dual core Celeron
2. About 1000 images for testing the app
3. Not a professional photographer
4. My primary use was to help identifying duplicate images, but the stack feature is confusing, even after the last upgrade. I have to [nuke my database to reindex photos](https://github.com/photoprism/photoprism/discussions/691#discussioncomment-200413). This is expectable for a software that is no stable yet, but it's midly bothering

**Comment by shotliff** [comment]
1. Docker container on local unraid server. Sever is a modified HP 290 with an i5-8400 and 16gb of ram. Instance is open to  the net with a reverse proxy using a Swag container. 
2. About 700+ photos and counting.
3. Amateur at most. 
4. My use case is using photoprism as back-up storage for cellphone photos. UX is 95% intuitive, it organizes things pretty well on it's own and makes it really easy to store/catalog/share things. Photoprism makes my photos feel a lot more useful than when they're disorganized on my phone.

**Comment by thomaseizinger** [comment]
1. Raspberry PI (currently model 3, upgrading to a 4B soon because the RAM is really limiting). I am **not** using the docker images because I like to manage these services using `systemd`. For example, using `systemd` I can declare that `photoprism` has a dependency on the mounted harddrive (which is just attached via USB) and as such will not start it unless the harddrive is plugged in and mounted.
2. About 10000 pictures and videos
3. Amateur
4a. I really like the "automate things as much as possible" aspect. I am hoping to use photoprism to easily gain "control" of my pictures (I plan to sync my phone pictures to it as well). This includes deleting pictures I am not interested in. I would be really nice if photoprism can make it easy to identify good or "worthwhile" pictures to keep.
4b. I also enjoy having full access to all my photos from my phone without having to store them there locally.
4c. Being able to share albums with my family is also important to me.

**Comment by ssonicmail** [comment]
1. Self hosting i7 4th Gen; photoprism running as a docker instance in Ubuntu VM  (Hyper V) with 4GB of RAM. Originals folder mounted via cifs on Ubuntu. It's a Windows server share running refs. I also use firewall for security and reverse proxy.
2. 9k pictures, 1k videos
3. Amateurs photographer, many pictures I store area just family and mobile phone snaps, some real camera pictures.
4. My use case is this: I tried piwigo, jellyfin and kodi but could never get it to work well, with large photos it was either slow and not good on mobile or iPad. Photoprism is great, features /characteristics I like the most:
- Speed - it handles 42mp pictures very well by creating a smaller version, it does it out of the box, therefore works well on wifi or via vpn.
- Compatibility with video files- can see all video formats I have from various cameras.
- Ability to handle existing folder organised collections. I can sync my photos to smb drive using another Android app and then refresh the library.
- Web interface do good, there's little need for mobile app.
- the fact that you can use Chrome on Android and share images via WhatsApp just like from Google photos app

Features I miss the most from other platforms:
- Multiple user support ( I know it's coming:)

Overall I think I can finally stop using Google photos which will soon to become a paid service and will likely loose some appeal.
Setup was quite easy as well, even though I have had little exposure to docker before.
Thanks to all the developers, such a great piece of software!

**Reply by zierbeek** [reply]
Could you maybe help me out a bit?
I have it running on a vm in a docker container with my photos mounted via cifs. I can't seem to get them imported. There is something wrong with my paths.

**Reply by lastzero** [reply]
We need more details to help you:
- How exactly did you mount which folders via CIFS? 
- Do you see any errors? 
- Did you check permissions?

**Comment by jpbuttet** [comment]
![Capture](https://user-images.githubusercontent.com/7593186/148243564-b04b766c-e554-4081-93df-54ee44275c3e.JPG)
Hello,
I just installed PhotoPrism yesterday.
PhotoPrism running on Docker on Ubuntu on HPE Proliant DL380P Gen8
(dual Intel(R) Xeon(R) CPU E5-2660 v2 @ 2.20GHz / 20 cores / 40 threads / 64 MB Ram)

PhotoPrism runs smoothly.

Just launched a quick run with some photos, (via Import feature in PhotoPrism GUI) 
I can just say that it's easy to setup and to use.

I am now copying a part of my photo/video library into the /originals folder via Webdav from a Mac to the server. The goal is to test PhotoPrism with about 20'000 files.

Already copied 17'000 files in 2h45 min. 
I've launched the indexing of the new files in parallel, PhotoPrism still indexing about 2500 files /hours.
With all theses concurrent tasks running, Photoprism in very responsive and I can browse photo without any delay.

This app is a marvel, and I glad to be a new sponsor. Many thanks for the work.

I've also tried to access PhotoPrism with my iPhone, and it works very well :-)


Best regards.
Jean-Pierre

**Reply by jpbuttet** [reply]
This is a reply to myself in order give an update and provide some information that could be useful to somebody else..

Initial Photoprism testing looked okay to me.
I just reset the database deleted the content of the originals folder.

Things now are getting serious ;-)  I mounted my 4TB (about 300k pictures) picture repository as external read only folder like this:

I put following line in my host's fstab:
https://xxx.xxx.xxx.xx/myserver/webdav /media/photoprismwebdav davfs ro,user,noauto 0 0

and I added following line in photoprism docker-compose.yml :

    "/media/photoprismwebdav:/photoprism/originals"

Photoprism is now indexing the whole repository ... (no need to copy nor import my originals) ;-)

**Comment by CaptnSeraph** [comment]
Currently hosting on AMD 3400g 4c/8t processor with 32GB RAM, NVME (system disk),  SSD (cache and database),  HDD (original storage) 

Currently hosting 144,000 pictures with plans to import another 100k once lightroom finally lets go of them (they have a tool to bulk download, but it doesnt work. Im never letting go of original storage again)

Im using it as a replacement for lightroom cloud. and access photoprism via phones, tablets and PC's

I am a professional digital artist and photo editor, and photoprism is being used for my reference material and archive.

I like that its quick to scan, generates thumbnails as per my specification and is customisable if i really wanted to dive into the code (had a look through the dev version and can see some changes i might make for my own personal use case) and most importantly... i dont have to spend 2 weeks to pull my photos back out as lightroom forces me to.

Happy to be a sponsor and heres to hoping you get enough support to keep this software competitive and put some bread on the table.

**Reply by harshavmb** [reply]
Hi @theseraphim,

Just a sheer mad question. I'm hosting Photoprism on Raspberry Pi 4 (SD card) with external WD elements HDD. I would like to move away from SD card -》NVMe SSD while still retaining external HDD for storing originals. 

Will photoprism benefit with this move? Reading photos, videos are okay. The painful part is the initial upload & indexing. Having checked on grafana, I see lot of I/O waits specially on the device mounted with SD card rather external HDD. 

Please have your say..

**Reply by CaptnSeraph** [reply]
If you can move the cache and the database to faster media you will see a speed boost, as to the pi4 CPU being used for the indexing, that's only about 20% as fast as the 3400g here so that may be your bottleneck.

I certainly would think it worth trying.

**Comment by dinoschachten** [comment]
I hope you don't mind my scenario explained in a bit of a story...

The problem:
As a photographer and filmmaker, I keep my 15-20 Terabyte of photo and video data on a NAS.
Originally I started using DigiKam to start tagging these 100,000ish files, with the database hosted on a Dell Precision Workstation (Ubuntu).
Turns out running DigiKam on a macOS client (using collections from the NAS via SMB) is very slow at both indexing and browsing (or loading) despite transfer speeds between client and NAS maxing out the Gigabit network.
More frustratingly, macOS Catalina has introduced to severe issues:
1. All network traffic appears to go through Finder which refuses to use more than one core on my (2012) i7 quad, bottlenecking the performance and resulting in other processes being affected and extreme fan speeds. 
2. The network connection occasionally drops out, resulting in DigiKam seeing empty collections, having to start indexing from scratch to find all the images again.

The search:
I was looking for a web-based solution that would be more centrally managed and easily accessible, not having to set up each client individually, and reducing the load on my (apparently somewhat unreliable) Mac client.
After looking into Piwigo for a company environment, I was happy about their Plugin system, but frustrated with their limitations that unfortunately go down to the core (lack of RAW support, poor support of "special" characters), but liked the generation of easily browsable thumbnails. So I wanted something similar, but supporting the file types any serious user of a photo management solution needs most: RAW formats (ironic to me that many photo management solutions don't support what can perhaps be considered the essence of digital photography).

The revelation:
During my research, PhotoPrism looked very intriguing from the start despite missing a few features I had put on my requirements list. What really convinced me is how rapidly the development progresses to implement new, powerful features, and the fact that AI is being utilised in several really useful ways - the time to implement and push these amazing technologies is definitely now, and I am excited that the team behind PhotoPrism has fully embraced that fact.
I finally finished my installation the day before yesterday (on the aforementioned Ubuntu workstation) and love the UI, the performance, the customisability. There are certainly kinks to work out to make it even better at what it does, but seeing what is already in the works and on the roadmap makes me absolutely confident about this choice.

**Comment by adambeck7** [comment]
- Ryzen 3900x with 32gb of ram. 1tb NVME cache and 50tb array running Unraid. 
- 50k photos and 1500 videos.
- Just a hobby.
- I love how fast it is and that it now has facial recognition. I would really love to see some more advanced editing features (and of course multi-user support :)

**Comment by BoKKeR** [comment]
8700k with 32gb of ram. 1.5tb NVME cache and 40tb array running Unraid.
38k photos and 1000 videos.
Just hobby to visualize my photos from different phones and cameras
Love the metadata extraction and wide range of supported formats!

**Comment by centralhardware** [comment]
1. proxmox running on ovh dedicated server (Intel Xeon-E 2136 ,64 GB ECC , 2xHDD). photoprism running inside ubuntu vm (4 vCPU, 6 GB RAM)
2. 37K photos
3. i'm java dev. i use photoprims to manage my home photo library.
4. in most i like dynamic development of the project.

I try photoprism somewhere a year ago (intel nuc i3). indexing was very slow and I wasn't impressed with recognition. recently decided to try again and was impressed with the project

**Reply by DrMemoryFish** [reply]
Hey,
I have a Windows machine, however, I can run a VM.
I need guidance on how to set everything up. 
All the docs and videos on youtube are either too technical for me to understand, or are outdated.
Could you please assist me? 

Thank you.

**Reply by alexisLefebvre** [reply]
> Hey, I have a Windows machine, however, I can run a VM. I need guidance on how to set everything up. All the docs and videos on youtube are either too technical for me to understand, or are outdated. Could you please assist me?
> 
> Thank you.

I suggest you to open a new discussion: https://github.com/photoprism/photoprism/discussions

It will have more visibility than this message, and the OP used Proxmox, not Windows, so they may not be able to help you.

**Comment by richardklose** [comment]
Selfhosted at home on an old Lenovo X230 (i7 / 16GB RAM) with external 2TB SSD
73k Photos / 3,5k Videos
Just hobby photographer privately, at youth camps we attend as volunteers and at soccer games of my brother.
I started using photoprism quite early during the development phase. Even though it was not stable back then, it still was better than most of the other solutions that could have been a good replacement for iCloud Photos. I wanted something to easily keep and browse photos from all our family devices. Also face recognition is 🚀 🔥! Most of our photos are from youth camps with a lot of people. Having a good way to tag and find people on such a large photo library is a game changer.

**Comment by harshavmb** [comment]
1) Self hosted at home on Raspberry pi 4B (SD card), 8TB WD elements external HDD. Using docker at the moment
2) Current photo collection at 15K (started 2 weeks ago), will add more going forward
3) Would like to have absolute data sovereignty. Photoprism is a nice tool to achieve as far as photos & documents are concerned (WebDAV)
4) I've no complains, so far it has been very stable. Waiting for multi-user support with ACLs so as to share with family members & friends. 

Would like to be a sponsor as well & share with my circle.

**Comment by blue-kaleidoscope** [comment]
1. Self hosted on a QNAP NAS [TS-451+](https://www.qnap.com/de-de/product/ts-451+) with 8 GB RAM
2. Current photo collection with around 67k photos and it runs super smooth(!)
3. In the past I considered myself as semi-amateur/professional and I heavily used Adobe Lightroom for my DSLR raw photos. Now I am a family father and I am only using the photo library for hobby purpose
4. What I love about photoprism

- I take pictures with roughly 4 devices containing family iPhones, compact camera and still sometimes SLR --> Photoprism combines them all in one place and when EXIF is properly set I get a nice history of events when multiple cameras were in use
- The whole import and visualize workflow is 99% automatic: Automated export of iPhone photos/videos to import folder of photoprism > every night I cron trigger the import AND convert task of photoprism > The next morning by latest I can enjoy all pictures and videos (for compact camera/SLR content I have to put the SD card manually into a card reader attached to the RaspberryPI and from there via a shell script they get transferred to the import folder, that is the missing 1% ;-) )
--> And that is the biggest difference to Lightroom 6 at least. The import and organization was always manually and sometimes error prone.

So, keep up the great work!

**Comment by NaXal** [comment]
Hello,

I use combination of NextCloud and PhotoPrism.

My PhotoPrism installation is within a Hyper-V VM running Ubuntu 22 with Docker and Docker Compose.

Host Hardware :- Intel Core i7 4790 / 32GB (8x4) DDR3 / 256GB SATA SSD / 1Gbps  (Dedicated for 24x7 Server Duty)
Hypervisor :- Windows 11 Pro Hyper-V
Guest VM :- Ubuntu 22 LTS / 8 Cores (Threads) / 4GB RAM
Docker Details :- Default YML Compose file but I have 5 for individual users with different ports (and off course password)
NginX Reverse Proxy :- To use different domains for each of these Docker instances for easy access.

Clients (users) are using Nextcloud and respective iOS or Andoird app for auto uploading their phone media (mainly camera app picts n vids). Each Docker instance of PhotoPrism connects to those respective Nextcloud account via WebDAV for importing.

Setup runs really smooth via the web app method as described in PhotoPrism website for usage.

Only issue I see is with Hyper-V, because initial webdav importing was dead slow (I have an [issue opened here](https://github.com/photoprism/photoprism/discussions/2692)).  Previously I ran PhotoPrism with Ubuntu VM VMWare but there I didn't face any such issue, but with Hyper-V something is broken. However, once the initial painful importing is complete, day to day usage experience is really smooth and polished !!

**Reason for Using PhotoPrism :-** The main idea is to take control of my own data. It all started with Google back stabbed us by pulling the plug on unlimited Google Photos storage. Initially I started with NextCloud. It's really good as backend however it lacks the needed feature to be a front end. Further searching led me to PhotoPrism, just exactly what I wanted and really happy to see WevDAV importing worked with NextCloud (well in my case, very slow though)

It's an excellent project, really helpful for people like me, who wishes to take back control of our own data from large corps.

Thanks.

**Comment by robra2** [comment]
Hello,

- self hosted on QNAP TS251A / 8GB
- Containerstation / docker, managed with Portainer
- Maria DB container (started with the QNAP Maria DB which also works)
- plan to use it for our home photo library 
- started with the first 3k pictures --> great, the remaining 27k will follow

...still searching for an efficient process to sort the pictures (I'm not made for deleting pictures :-D)

**Comment by huangda1982** [comment]
- old i5 laptop 8G ram
- 20 000 pictures, 2000 for test now
- interested in these features: AI tagging, facial recognition, image quality assessment (artisticly), AI image beautification, daily photo memories
- current status: Still testing. A lot of faces are recognited, most of them are not clustered. CPU usage = 0%. Most people with glasses are not recognited.

**Comment by sunbeam60** [comment]
Rekindling this thread to celebrate the new release:

Small Dell 7000 Micro acts as home server: https://imgur.com/a/NZDdGAk

- Intel i7-12700T
- 32 GB RAM
- All photos stored on NAS, mounted via NFS on the server (Ubuntu Server 22.04 LTS)
- All PhotoPrism cache & storage on a 2 TB M.2 SSD, mounted into PhotoPrism docker folder
- ~64000 photos, ~4000 videos - the combined family album since 1998
- Upload photos via PhotoSync over FTP, into PhotoPrism's import folder. Overnight import running via cron. Sync happens whenever we come back home, or whenever we plug in to charge after 22:00.
- PhotoPrism running in a Docker container behind nginx reverse proxy for certificates. 

The thing we love most about it is:

- It works! We previously used Plex' in-built photo uploader and it basically never worked. Dreadful and caused a lot of quarterly dumps from phone, with associated duplicate finding etc. It was a real chore. PhotoPrism + PhotoSync just works.
- Speed! Scrolling through photos is bullet train quick.
- Hardware transcoding of videos! Suits Intel QuickSync particularly well.
- (Recently released:) Shared albums! We have family abroad and sharing photos with them used to happen via a posthaven blog. Now we can just add photos to a shared album link they already have.
- HEIC support! Lovely to have originals in full fidelity, with rapid JPEG previews.
- Web App! Super easy to share with family and friends; just a URL.
- Docker/self-hosting! Great to have our home media in full privacy.
- Face recognition! We have four kids, so it's fantastic to be able to get person-specific time lines and "see them grow up" in picture form.
- Places! We've traveled to a lot of places and it's amazing to easily find the pictures from "the second time we were in that country".

What we would love next:

- Video streaming, instead of waiting for transcoding.... or at least transcoding progress - for non-technical family members, the delay between transcoding start and video start can be confusing for them, as they don't understand what's happening.
- Search by date range - currently you can choose pictures in a month, but some time events break month boundaries.
- Automatic image importing - I know this is seen as an "impossible feature", but it would be great if PhotoPrism could monitor for folder changes and auto import.
- ActivityPub support for Albums. Would be awesome if the wider fediverse could be made aware when new photos appear. 

Thanks once again for creating an awesome, self-hosted photo management app! It's a pleasure to be a paid supporter.

**Comment by ippocratis** [comment]
Selfhosting on a raspbrrrypi-400 with docker-compose 

```
uname -a

Linux sway 5.15.78-1-MANJARO-ARM-RPI #1 SMP PREEMPT Sat Nov 12 17:13:38 UTC 2022 aarch64 GNU/Linux
```

~75000 photos on an external hard drive  mostly taken with family mobile phones 

Reverse proxying and requesting client certificates auth with caddy 

Mainly viewing photos on mobile devices with the unofficial [photoprism-android-client](https://github.com/Radiokot/photoprism-android-client) and the pwa if I want to look at metadata

Backing up to import folder over webdav with foldersync

**Comment by dankasak** [comment]
I'm self-hosting 3 instances ( me, my partner, my daughter ) on a high-powered home server:
- AMD Ryzen 5 7600X 6-Core Processor
- 32GB RAM
- 2x SSDs in RAID0

I have a simple backup script that runs every night for each instance, that:
- backs up the database
- synchronises database dump and all /originals and /storage to storj ( https://www.storj.io/ ) using rclone

I'm using apache virtual hosts to proxy different endpoints into each instance, and letsencrypt for SSL certificates.

All up I have around 0.5TB of photos and videos, with about 70,000 individual files. It works really well :)

What I like the most:
- AI face detection is great - it makes it much easier to migrate to and use
- The UI is great, and works really well on the mobile

What I'd like to see:
- A different kind of "Moments" grouping, based on the time and/or location a photo was taken. Use-case: I have photos from up to 20 years back, and I'd like to see things like holidays and daily outings automatically added to their own "moment".
- A sync method that didn't suck or cost money. All sync apps either 1) suck, or 2) are littered with ads, or 3) are commercial. Syncing photos is a part of the core experience.

What I'd like to work on:
- The "Moments" idea above. I've looked at the existing Moments code, and I think I can pull off another implementation. Maybe.
- TiDB ( MySQL compatible, distributed database ) support - I've already reached out via https://github.com/photoprism/photoprism/discussions/3218
- Postgres support - perhaps after I've figured things out by adding TiDB support, I can tackle this - I imagine it would be significantly more difficult

**Reply by sunbeam60** [reply]
I do agree that sync is an integrated part of photosync ... but holy hell it's a big effort. Plex tried for years to get reliable sync and failed miserably. It's just really hard in iOS ... only PhotoSync, yes a commercial offering, seems to work reliably.

**Reply by BoKKeR** [reply]
I use iCloud with iCloudPD container to download the photos from iCloud. Used photo sync before on android

**Comment by jojo-gfx** [comment]
At time only as backup solution, because the options for using it as Platform to make my photos available for others are not really implemented at time.
The still not existent possibility of make albums free as a gallery for not logged users is a big point. At time I only can give special links outside, but the photoprism itself has no option for that usage.
Also seems I can't use the map feature for people without a login ... not nice, because I was buy it especcially for that use... so at time it is relative useless here.

Hopefully there will coming more features for public access to use it as web-gallery and not only as backup solution for myself.

**Reply by lastzero** [reply]
First of all, we are super transparent and nothing has changed as you seem to suggest - please take the time to read our FAQ before coming to wrong conclusions:

> This project is about freedom and privacy, but not necessarily about free beer. We believe that it would be a mistake to state that there will be no costs, because clearly we have huge expenses, your server hardware has a price tag, and then you might want some additional features that need to be developed.

👉 https://www.photoprism.app/oss/faq

Secondl, our Community Edition isn't just a simple demo, only the commercial maps are missing. This is because, of course, they are commercial. A membership is just 2 EUR per month though and we also offer lifetime options.

Finally, you complain that we don't work fast enough, for example with a larger team, to release the features you want. How should this be possible if it's not worth paying for a membership, as you suggest? Our members are funding this. They are the only reason this software exists in the first place. Think about that for a minute. Thank you!

**Reply by jojo-gfx** [reply]
Sorry, was correct myself.
No, also the multiuser functions are in the other version only available and would think also the option for public galleries (or later?)?
Have had no problem to buy the map feature. This way is really ok, but I'm absolute allergic on subsriptions.
I never had a problem to support any things per donation or buy for special features, because, this is a fair way, but subscriptions are the method for the big concerns, so it isn't a good Idea for OSS.

Sorry for that words, but pressed against a wall I don't feel good anymore.

More important:
For me the absolute simplest basic functionality of a photoalbum is not implemented: To show my galleries in a simple way to unknown viewers (not only per direct link ... this is only for special cases usable). 

The software is so so extremely nice ans usable, but only by logged in person. ;(

**Reply by lastzero** [reply]
Then go for a lifetime membership or support us with your time so we can give you something in return. Complaining publicly is the wrong way to get what you want.

**Comment by anarcat** [comment]
> What infrastructure are you using?

Home lab server, [specifications](https://anarc.at/hardware/server/marcos/), but shared with many other services, on a home cable connection (250/50mbps).

> How many photos/videos do you manage with it?

According to Photoprism, 36243 photos and 625 videos (just imported those today).

> Are you a professional photographer or is it your hobby?

Definitely hobby, and less and less so unfortunately, lack of time.

> What do you like most about the app?

- it *generally* works as expected
- publicly shared folders
- support for video and "play live" photos (I didn't even *know* those even existed before photoprism, wild no?)
- compact, "gapless" view that crams a lot of images in a single page
- "web app" that works well on mobile (although I do prefer to use a mobile app because of authentication and, ideally, caching)
- fast viewer

**Comment by OzRocky** [comment]
**What infrastructure are you using?**

PhotoPrism on Docker on a Ubuntu VM on Hyper-V on a Dell PowerEdge R720. SSD for database and rust for images. 

It's actually a bit slow, but that might be because of all the layers of virtualisation.

**How many photos/videos do you manage with it?**

About 150,000 images

**Are you a professional photographer or is it your hobby?**

Hobby

**What do you like most about the app?**

Face recognition is probably the best feature. About 200 faces identified so far with another 700 to try to put a name to.

**Comment by tomasenskede** [comment]
_What infrastructure are you using?_

PhotoPrism LXC running (on proxmox) with 8 GB RAM, 6 cores, 2 TB M.2 plus 20TB NAS for images.

_How many photos/videos do you manage with it?
_
Lots... 

_Are you a professional photographer or is it your hobby?_

Amateur photographer

_What do you like most about the app?_

Album share, index, AI detection, face recognition, easy/good web gui for new users

Missing TV-apps...

**Comment by artincube** [comment]
Hello there.
I have been using **Photoprism** for 4 years maybe.

_What infrastructure are you using?_
I tried a few things but now all is working real well on an **M1 Max Laptop, With Docker**. It is my main machine. I have my Originals on an external RAID10 (+ 3 backups, 2 offsite) and the Master Library on an NVMe TB4 external. I often run two libraries at once on two containers, the other one (with the same items in the library but smaller and jpeg) entirely on the laptop, much smaller footprint.

_How many photos/videos do you manage with it?_
About **100k in the Library,** not finished archiving I've got sessions of thousands to account for.
Quite a few of these are very high resolution **16 bit 100-1000MPixels** so I give Docker lot of Ram (24GB) so all could be indexed.
Lot of videos also, 8000.

_Are you a professional photographer or is it your hobby?_
It is part of my professional/Art practice, a **tool**. Maybe it is a **therapy**.

_What do you like most about the app?_
I loved the **simplicity** and **no nonsense GUI** from day 1. I wanted **self-hosted** semi-diy semi-pro. I learned to love what is happening under the hood and developed **immense respect** for the work and dedication of those involved.

**Comment by annomatik** [comment]
- What infrastructure are you using?
Self hosted, Docker on a Synology NAS with MariaDB

- How many photos/videos do you manage with it?
Currently, about 3000, but that's just photos from the last couple of years. Eventually, I will add more, I've been using DSLRs for about the last 20 years and have now also discovered drone photography for me.

- Are you a professional photographer or is it your hobby?
Hobby, but a serious Hobby. :-)

- What do you like most about the app?
That I can self-host it on my NAS, the potential it has and that it gives me a nice map-view of the GPS data in my photos. I use it as a "private viewer / browsing" app, in addition to public Facebook albums, private WhatsApp group postings and my Lightroom catalogs.
I also enjoy the AI features (subject / situation / people detection), but they don't really work well for me and I apparently cannot even add manual faces if the AI doesn't detect anything.
